test/kickstart-templates/kickstart-centos.ks.template

```diff
@@ -2,7 +2,7 @@
 %include /main-network.cfg
 %include /main-cdrom.cfg
 
-%post --log=/var/log/anaconda/post-install.log --erroronfail
+%post --log=/dev/console --erroronfail
```

Comment:
I'm curious if lack of `--log` would mean the same, i.e. log to console

Reply:
Kickstart docs do not specify what happens when `--log` option is skipped.
https://docs.fedoraproject.org/en-US/fedora/f36/install-guide/appendixes/Kickstart_Syntax_Reference/#sect-kickstart-postinstall

Let me test it asap.

Reply:
When having the `%post --erroronfail` directive in a kickstart file without `--log` options at all, the console does not display logs.
```
[  218.202922] anaconda[2450]: Running post-installation scripts
[FAILED] Failed to start User Manager for UID 0.
[  287.959509] anaconda[2450]: .
[  287.959933] anaconda[2450]: Installation complete
```

Instead, they go to `/var/log/anaconda/ks-script-XXXXXXXX.log` files.
```
# ls -l /var/log/anaconda/
total 5420
-rw-------. 1 root root   36396 Mar 11 10:47 anaconda.log
-rw-------. 1 root root    3334 Mar 11 10:47 dbus.log
-rw-------. 1 root root    1985 Mar 11 10:47 dnf.librepo.log
-rw-------. 1 root root     120 Mar 11 10:47 hawkey.log
-rw-------. 1 root root 3424810 Mar 11 10:47 journal.log
-rw-------. 1 root root       0 Mar 11 10:47 ks-script-8kawzpgg.log
-rw-------. 1 root root      63 Mar 11 10:47 ks-script-b6rxlp8a.log
-rw-------. 1 root root      40 Mar 11 10:47 ks-script-i44z8rpm.log
-rw-------. 1 root root  122545 Mar 11 10:47 ks-script-ofgfu0nf.log
-rw-r--r--. 1 root root   26706 Mar 11 10:47 lorax-packages.log
-rw-------. 1 root root  224774 Mar 11 10:47 lvm.log
-rw-------. 1 root root  167238 Mar 11 10:47 packaging.log
-rw-------. 1 root root  168279 Mar 11 10:47 program.log
-rw-------. 1 root root  177348 Mar 11 10:47 storage.log
-rw-------. 1 root root 1166609 Mar 11 10:47 syslog
```

Reply:
Ack